processor/processor.go

```diff
@@ -1727,6 +1733,17 @@ func (proc *Handle) processJobsForDest(partition string, subJobs subJob) *transf
 	reportMetrics = append(reportMetrics, validatedReportMetrics...)
 	// TRACKING PLAN - END
 
+	for sourceID, events := range groupedEventsBySourceId {
+		if len(events) == 0 {
+			continue
+		}
+		source, _ := proc.getSourceBySourceID(string(sourceID))
```

Comment:
Curious, why don't we skip if we didn't get the `backendconfig.SourceT`, because we will be sending stats for empty `sourceId`, `sourceType`, `workspaceId`?

Reply:
Is it just to capture the status?

Reply:
I have address this